ArchiSteamFarm/CardsFarmer.cs

```diff
@@ -347,16 +376,36 @@ internal sealed class CardsFarmer : IDisposable {
 
 				float hours = 0;
 
-				Match match = Regex.Match(hoursString, @"[0-9\.,]+");
-				if (match.Success) {
-					if (!float.TryParse(match.Value, NumberStyles.Number, CultureInfo.InvariantCulture, out hours)) {
+				Match hoursMatch = Regex.Match(hoursString, @"[0-9\.,]+");
+				if (hoursMatch.Success) {
+					if (!float.TryParse(hoursMatch.Value, NumberStyles.Number, CultureInfo.InvariantCulture, out hours)) {
 						Logging.LogNullError(nameof(hours), Bot.BotName);
 						return;
 					}
 				}
 
-				GamesToFarm[appID] = hours;
+				games.Add(new Game {
+					AppID = appID,
+					HoursPlayed = hours,
+					CardsRemaining = cardsRemaining
+				});
+			}
+
+			IEnumerable<Game> gamesToFarm;
+			switch (Bot.BotConfig.FarmingOrder) {
```

Comment:
You should do sorting the moment before ```KeepFarming = NowFarming = true;``` instead of at the end of ```CheckPage()``` - the order will be wrong if you have more badge pages than 1.